src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -372,7 +372,7 @@ private void OnScrollAnimationEnded(object sender, ScrollEventArgs e)
             onAnimation = false;
             if (onAlignAnimation) {
                 onAlignAnimation = false;
-                PageAdjust(e.Position.Y);
+                if (loopEnabled) PageAdjust(e.Position.Y);
```

Comment:
could you make this as like this?

if (false  == xxx)
{
&#x2001;blah blah;
}


you can use auto code editing by "ctrl+k+d" or "ctrl+k+f" in Visual Studio.

could you check the sites below for easy way of editing code?

- Visual Studio in Windows : https://stackoverflow.com/questions/1435691/visual-studio-format-entire-file

- VS Code in Ubuntu : https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Reply:
Thank you for reviewing with guides.